libs/state/spec/rx-state.service.spec.ts

```diff
@@ -590,5 +594,18 @@ describe('RxStateService', () => {
       state.hold(of(1, 2, 3), effect);
       expect(calls).toBe(3);
     }));
+
+    it('should invoke provided ErrorHandler', fakeAsync(() => {
+      const state = setupState({ initialState: initialPrimitiveState });
+      const customErrorHandler: ErrorHandler = {
+        handleError: jest.fn()
+      };
+      TestBed.overrideProvider(ErrorHandler, { useValue: customErrorHandler });
+      setCurrentInjector(TestBed.inject(Injector));
```

Comment:
We shouldn't be using this inside tests. Just remove the `inject` function and use the constructor injection as @edbzn proposed; therefore, you won't need to set the current injector.